src/exercises/Exercises.js

```diff
@@ -34,6 +35,7 @@ export default function Exercises({
   articleID,
   backToReadingAction,
   keepExercisingAction,
+  backToArticleAction,
```

Comment:
Emma, I still don't like that we have both `backToReading` and `backToArticle`. 
Can we only keep one? 
In my case I'd be happy to only have `backToArticle` be used in both the cases.
I guess for you it's the same?

Reply:
Yeah in our case it's the same button. I'll change it to backToArticle in both cases then.

Reply:
Oh wait. We can't do that. BackToArticle is only possible when it is the exercises for a specific article. So "backToReading" is needed when they enter through zeeguu.org/exercises. 

But I guess we can still do it so Exercises only have two props: backToReadingAction and keepExercisingAction - and the functionality of backToReadingAction is then either backToArticle or backToReading depending on if they come from ExercisesForArticle or just Exercises. Will that be okay?